src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -150,7 +150,7 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
   }
 
   requiresVideoElementWorkaround(): boolean {
-    return this.isSafari();
+    return this.isSafari() && this.majorVersion() <= 12;
```

Comment:
Checking since we do not officially support Safari version less than 12, should this be `== 12`?

Reply:
Originally i have it === 12 but change it to this to be more general.